app/main.py

```diff
@@ -1,3 +1,21 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name: str,
+                 description: str,
+                 weeks: int | float) -> None:
```

Comment:
Here it would be better to make it in one line, or every argument on a new line

```
    def __init__(
        self, 
        name: str,
        description: str,
        weeks: int | float
    ) -> None:
```